src/domain/entities/DhisDataPackage.ts

```diff
@@ -45,7 +45,7 @@ export type EventsAPIResponse = Omit<EventsResponse, "instances"> & {
     events?: Event[];
 };
 
-type Coordinates = [number, number];
+export type Coordinates = [number, number];
```

Comment:
Now that we are refactoring it, let's take the opportunity to tag the coordinates type: `[latitude: number, longitude: number]` (or the other way around! not sure, that's why being tagged is nice :))